### PR TITLE
have build script clean up JSON formatting of `dpd-formatted.json`; add `build` script to `package.json`; document in README

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -6,7 +6,8 @@ const outputFile = path.join(__dirname, '..', 'dpdb.json');
 const formattedJSON = fs.readFileSync(inputFile, 'utf8');
 
 // Use JSON.stringify/parse to ensure that the JSON is valid
-// rather than just stripping out space/newline chars
-const minifiedJSON = JSON.stringify(JSON.parse(formattedJSON));
+// rather than just stripping out space/newline chars.
+const formatify = (str, indent=0) => JSON.stringify(JSON.parse(str), null, indent);
 
-fs.writeFileSync(outputFile, minifiedJSON);
+fs.writeFileSync(inputFile, formatify(formattedJSON, 2));
```
Comment (jsantell) [inline scripts/build.js]: I'd maybe check to see that if the file is improper JSON, it does not overwrite anything (I think it should just throw and stop, though)

**Review by jsantell — APPROVED** [review]
Looks good! Now I can imagine in the future some simple validations for these entries..

Travis test failing but there's no tests so that's fine

File: scripts/build.js (hunk 1 above)
Comment (cvan) [reply scripts/build.js]: good call, yeah

**Comment by cvan** [comment]
> Looks good! Now I can imagine in the future some simple validations for these entries..

filed issues #1 and #2 👍 

> Travis test failing but there's no tests so that's fine

tests passing now :1st_place_medal:

File: scripts/build.js [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+try {
+  dpdbObj = JSON.parse(dpdbStr);
+} catch (err) {
+  throw new Error(`Could not parse as JSON from file "${inputFile}"`);
+  console.error(err);
+  process.exit(1);
+}
 
-fs.writeFileSync(outputFile, minifiedJSON);
+if (argv.includes('--write') || argv.includes('-w')) {
```
Comment (cvan) [inline scripts/build.js:18]: thoughts on this? I didn't want to get too fancy just yet

Comment (jsantell) [reply scripts/build.js:18]: Yeah I think this looks great, and makes a quick and easy test too

Comment (cvan) [reply scripts/build.js:18]: ok, sweet - thanks

**Pull request merged**